create-rust-app_cli/src/content/project.rs

```diff
@@ -80,12 +80,12 @@ fn add_bins_to_cargo_toml(project_dir: &std::path::PathBuf) -> Result<(), std::i
     let append_to_toml = format!(
         r#"
 [[bin]]
-name = "fullstack"
-path = ".cargo/bin/fullstack.rs"
+name = "run_yarn_fullstack"
+path = ".cargo/bin/run_yarn_fullstack.rs"
 
 [[bin]]
-name = "tsync"
-path = ".cargo/bin/tsync.rs"
+name = "run_yarn_tsync"
+path = ".cargo/bin/run_yarn_tsync.rs"
```

Comment:
8.1.0 doesn't run tsync via yarn anymore nor does it use the global tsync binary.

Since you can uninstall tsync via `cargo uninstall tsync` and use `cargo tsync` normally, do you think this resolves the issue?